op-node/cmd/batch_decoder/main.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-node/cmd/batch_decoder/fetch"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/urfave/cli"
+)
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "batch-decoder"
+	app.Usage = "Optimism Batch Decoding Utility"
+	app.Commands = []cli.Command{
```

Comment:
Nit: the other `op-node/cmd` packages provide sub-commands that plug into the actual op-node command. If it's not an `op-node` sub-command, maybe it should live in a different package? Like a new root level one, or maybe just part of `op-wheel` (the other stand-alone CLI utils)

Reply:
Stateviz is its own binary. I don't really want to make it an op-node subcommand. I am open to potentially moving it to a new location, but I went with this to keep it simple.